### DISCUSSION TITLE
Does Github Pro include Github copilot for free?

### DISCUSSION THREAD

**Discussion by jorgeas80** [community/community#37783] [GitHub Education]
Hello,

I've been using Github Copilot for a while, using a paid account (10$/month). Now, I've been accepted as a student of Github Global Campus, which includes a free Github pro account, as I checked in my profile.

My question: does this pro account also include copilot for free, or will I still need to pay for it?

Many thanks in advance

**Comment by DenverCoder1** [comment]
Verified students get access to Copilot for free.

You can view more details on billing for Copilot here: https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot

You can also check the status of your subscription here: https://github.com/settings/billing

**Comment by jorgeas80** [comment]
Hi @DenverCoder1, thanks for your answer

I've been paying Copilot since September, but I've been accepted into Global Campus past week. And I still see 10$/mont on my billing page. Am I going to be charged again next month for Copilot? 

![image](https://user-images.githubusercontent.com/772448/199066603-516ce0af-9272-49a9-9f26-169666b05ffb.png)

I wrote an email to support at githubsupport dot com, and they just told me to, I quote: _contact the partner directly by clicking the help available at link on the offer tile_

You are the partner, Github. That email was written precisely for that reason. And now I'm in an infinite loop

**Reply by DenverCoder1** [reply]
Does anything appear in the Edit menu? Maybe you need to cancel copilot to get it to be free, or it might be fixed at the next billing cycle?
